spec/spec_helper.rb

```diff
@@ -14,7 +14,7 @@
 
   def assert_correct_indenting(extension='rb', string)
     whitespace = string.scan(/^\s*/).first
-    string = string.split("\n").map { |line| line.gsub /^#{whitespace}/, '' }.join("\n").strip
+    string = string.split("\n").map { |line| line.gsub(/^#{whitespace}/, '') }.join("\n").strip
```

Comment:
Ahh. I think I get it. This is trying to do what `<<~EOF` does I think. It seems to strip the first indent from the rest of the line.